client/components/web-preview/index.jsx

```diff
@@ -138,7 +138,7 @@ export class WebPreview extends Component {
 			return;
 		}
 
-		if ( ! this.iframe ) {
+		if ( ! this.iframe || ! this.state.iframeUrl ) {
```

Comment:
Since this function is the only one ever setting `iframeUrl` state to anything other than a falsey value, won't this always abort?